packages/vscode-extension/src/treeview/account/accountTreeViewProvider.ts

```diff
@@ -110,7 +110,8 @@ async function azureAccountStatusChangeHandler(
       envTreeProviderInstance.refreshRemoteEnvWarning();
     }
   } else if (status === "SigningIn") {
-    instance.azureAccountNode.setSigningIn();
+    // "Azure Account" extension only sends SigningIn signal without SignededOut in 0.10.x, so remove this status change until it's fixed.
```

Comment:
Since we don't block user after user clicks 'Sign in to Azure' now, user may click multiple times before logged in. Have you tested related scenarios to make sure this won't introduce other issues?

Reply:
Good point, thanks. Users will be blocked by modal prompt and if they click during signing in, it will start a new progress.